rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.53"
+channel = "stable"
```

Comment:
I'd like this to still be pinned to a specific version. It only requires a bump every 6 weeks and with a pinned version, we get more assurance that "checkout and compile" just works.